Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM docker
 
-RUN apk add parallel
+RUN wget https://github.com/keboola/keboola-as-code/releases/download/v2.12.1/keboola-cli_2.12.1_linux_amd64.apk
+RUN apk add --allow-untrusted ./keboola-cli_2.12.1_linux_amd64.apk
```

Comment:
Version by si mohol dat do nejakeho ARG/ENV pre jednoduchsi update.

Reply:
BTW: `alpine` je priamo podorovane, tak by bolo lepsei pouzit oficialny sposob ako `--allow-untrusted`

https://developers.keboola.com/cli/installation/#alpine

Reply:
Můžeš poradit, jak to udělat? Jakože kde uložit ten env var a jak ho dostat sem

Reply:
Verzia sa definuje takto `apk add packagename=1.2.3-suffix`

Reply:
> Můžeš poradit, jak to udělat? Jakože kde uložit ten env var a jak ho dostat sem

Myslel som len tu v `Dockerfile`, aby to tam nebolo 2x, napr.
```
ARG CLI_VERSION="..."
```

Dalej to pouzivas ako ENV `$CLI_VERSION`.

Pri `docker build` sa da arg definovat ina hodnota, inak bude mat tento default.